js/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "include": [
+        "src/index.js"
+    ],
+    "compilerOptions": {
+        "declaration": true,
+        "emitDeclarationOnly": true,
+        "allowJs": true,
+        "checkJs": true,
```

Comment:
1. Have you tried `"strict": true`? There are probably lots more errors with it on, though.
2. Since somewhere around 4.0, you only need `checkJs: true` since it now implies `allowJs: true`.

Reply:
Absolute +1 to making TS strict. A must to detect all kinds of issues.

Reply:
Cool, I'll add `strict: true` and see how it goes.

Reply:
Yeah, just way too strict for me right now. :) Will look at that later.